### PR TITLE
[TASKMGR] Favor _countof(), no functional change intended

### PR DESCRIPTION
**Pull request by JoachimHenze**
Taskmgr used a mixture: Sometimes _countof(), sometimes ARRAYSIZE() and sometimes it calculated the count plain by sizeof(var)/sizeof(TYPE). Harmonize everywhere to _countof() as it is the shortest solution.

Fix some formatting sins, like placing comments before else-statement. Shorten the length of some very long line
intentionally *without introducing additional linebreaks* !

Shorten vertical length of some functions to increase their chance to fit on the screen without scrolling.

Fix wrong indentation level in TaskManagerWndProc().

No functional change intended.

JIRA issue: none

### CODE REVIEW

File: base/applications/taskmgr/applpage.h [hunk 1]
```diff
@@ -9,17 +9,15 @@
...
+extern HWND hApplicationPage;               /* Application List Property Page */
+extern HWND hApplicationPageListCtrl;       /* Application ListCtrl Window */
+extern HWND hApplicationPageEndTaskButton;  /* Application End Task button */
+extern HWND hApplicationPageSwitchToButton; /* Application Switch To button */
+extern HWND hApplicationPageNewTaskButton;  /* Application New Task button */
 
 INT_PTR CALLBACK    ApplicationPageWndProc(HWND hDlg, UINT message, WPARAM wParam, LPARAM lParam);
 void                RefreshApplicationPage(void);
 void                UpdateApplicationListControlViewSetting(void);
-
```
Comment (binarymaster) [inline base/applications/taskmgr/applpage.h:22]: I'd keep this one.

File: base/applications/taskmgr/taskmgr.c [hunk 2]
```diff
@@ -868,11 +859,8 @@ void SaveSettings(void)
 
 void TaskManager_OnRestoreMainWindow(void)
 {
-    //HMENU hMenu, hOptionsMenu;
     BOOL OnTop;
 
-    //hMenu = GetMenu(hMainWnd);
-    //hOptionsMenu = GetSubMenu(hMenu, OPTIONS_MENU_INDEX);
```
Comment (binarymaster) [inline base/applications/taskmgr/taskmgr.c:875]: Hmm... this was something [2]

**Review by binarymaster — APPROVED** [review]
Looks fine to me, perhaps Hermes would want to review this.

File: base/applications/taskmgr/applpage.h (hunk 1 above)
Comment (JoachimHenze) [reply base/applications/taskmgr/applpage.h:22]: Ok, will do.

**Pull request merged**